WordPress/gutenberg
# Mobile: Dashicon color override

## Description

This PR adds the possibility to override the default Dashicons color. This is necessary to solve https://github.com/wordpress-mobile/gutenberg-mobile/issues/546 .

I didn't find any already implemented way to do this, so I had to modify a couple of Gutenberg's `index.js` files. I'm not sure if this will affect Gutenberg Web, but local tests were passing.

Hopefully there's a better way!

![img_1268](https://user-images.githubusercontent.com/9772967/53107540-80166180-3535-11e9-93a2-05cb51e7d2e7.png)

**To test:**
- Run the app.
- Check that the add block button is filled with `$bule-wordpress` color.
